src/avt/Expressions/General/avtMagnitudeExpression.h

```diff
@@ -37,6 +37,10 @@ class     vtkDataArray;
 //    Hank Childs, Mon Jan 14 17:58:58 PST 2008
 //    Allow constants to be created as singletons.
 //
+//    Eric Brugger, Thu Dec  8 15:03:57 PST 2022
+//    I removed the DeriveVariableVTKm method since vtkmExpressions is
+//    no longer supported.
```

Comment:
The `vtkmExpressions` code hasn't been used in about 4 years when I switched to using vtkh instead of vtkm directly. I don't recall where `vtkmExpressions` was defined, it may have been in vtkm or it may have been in an interface library I created. It's definitely not there anymore.